core/src/main/java/org/stellar/anchor/sep31/Sep31Transaction.java

```diff
@@ -96,12 +101,12 @@ public interface Sep31Transaction {
 
   String getRequiredInfoMessage();
 
+  void setRequiredInfoMessage(String requiredInfoMessage);
```

Comment:
This was missing? Wow, thanks for catching.

Reply:
It wasn't missing but was disorganized in another part of the file. What I did was group the get/set together for this variable, the same way the other getters and setters are grouped.